docs/build/isc/v1.1/docs/tutorials/defi-yield-farming.md

```diff
@@ -0,0 +1,422 @@
+---
+description: Yield Farming Tutorial Using Solidity.
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - EVM
+  - tutorial
+  - defi
+  - yield-farming
+---
+
+import DeployAdmonition from '../_admonitions/_deploy_a_smart_contract.md';
+
+# Yield Farming DApp with Pyth Network Price Feeds
+
+## Introduction
+
+In this tutorial, we will guide you through the process of building a basic yield farming decentralized application (DApp) using [Solidity](https://docs.soliditylang.org/en/v0.8.26/), one of the most widely adopted programming languages for [Ethereum](https://ethereum.org/en/) smart contracts, and the [Hardhat](https://hardhat.org/) development environment, a powerful and flexible framework for building on the Ethereum blockchain. Additionally, we will use the Pyth Network Price Feeds to retrieve the IOTA/USD price feed, which will be used to calculate rewards for stakers.
+
+## What is Yield Farming
+
+Yield farming, also known as liquidity mining, is a way for users to earn passive income by providing liquidity to a blockchain protocol. This involves depositing assets into a protocol, which in return offers rewards in the form of fees, interest, or other incentives. For instance, you might supply liquidity to a decentralized lending platform like [Aave](https://app.aave.com/) or a decentralized exchange using an Automated Market Maker (AMM) model like [Uniswap](https://uniswap.org/). In return, you can earn rewards.
```

Comment:
I don't think we have any affiliation with these, could you rephrase it without including their names and websites?

Reply:
Done, thanks